src/_posts/hands-on-ii.md

```diff
@@ -0,0 +1,177 @@
+
+```html
+<a-entity id="left-hand"
+          static-body="shape: box"
+          vive-controls="hand: left">
+</a-entity>
+<a-entity id="right-hand"
+          static-body="shape: box"
+          vive-controls="hand: right">
+</a-entity>
+```
+
+Each hand is actually an entity which has two components `static-body` and `vive-controls`. The first is required by the physics engine to detect when it is colliding with other bodies. The second shows the specified remote making it automatically match the position and rotation of the tracked control. Components can be one-value such as the former `hand-controls` or multi-value like `static-body` or `vive-controls` (i.e., comprised of several named values like the _mass_ or the colliding _shape_). The different values of a multi-value component are called properties.
+
+Finally, systems are pure behaviour extensions and therefore, they are usually invisible. Take the physics engine for instance. It is a system and it orchestrates the interactions between those entities with `static-body` and `dynamic-body` components (among others).
+
+### A note on primitives
+
+[Primitives are entities under the hood](/docs/0.6.0/introduction/html-and-primitives.html#primitives). Primitives have specific attributes mapped to certain properties of components implied by the primitive. For instance:
+
+```html
+<a-cylinder id="suction-pad"
+            position="-0.2 0.025 0.286" rotation="-90 0 0"
+            height="0.1" radius="0.005" color="red">
+</a-cylinder>
+```
+
+This `a-cylinder` primitive implies [`geometry`](/docs/0.6.0/components/geometry.html) and [`material`](/docs/0.6.0/components/material.html) components. The `primitive` property of the `geometry` component is automatically set to `cylinder`. Then, `height`, `radius`, and `color` are attributes mapped to `geometry.height`, `geometry.radius`, and `material.color` respectively, while `position` and `rotation` are components on their own.
+
+The equivalent entity looks like:
+
+```html
+<a-entity id="suction-pad"
+          position="-0.2 0.025 0.286" rotation="-90 0 0"
+          geometry="primitive: cylinder; heigh: 0.1; radius: 0.005"
+          material="color: red">
+</a-entity>
+```
+
+## The A-Frame registry
+
+The [A-Frame registry](/aframe-registry/) is a centralised source of components. You can find a collection of components here and download the libraries or visit their home pages. Look at the [`aframe-auto-detect-controllers`](https://www.npmjs.com/package/aframe-auto-detect-controllers-component) component for instance. Given that I replaced the hand controls with the HTC Vive remotes, I could have used this component to automatically detect which particular tracked controller to use, e.g. HTC Vive or Oculus Touch.
```

Comment:
You use `controllers` instead of `remotes` above. Better to use a consistent nomenclature